dashboard/app/reporting_email.go

```diff
@@ -552,4 +561,23 @@ func appURL(c context.Context) string {
 	return fmt.Sprintf("https://%v.appspot.com", appengine.AppID(c))
 }
 
+func separateCC(to, cc []string) ([]string, []string) {
+	if len(cc) == 0 {
```

Comment:
For optimization purposes we could check `|| len(to) == 0` here as well b/c I think that if to is empty, we don't change cc as well. However, the generic code below should handle both cases correctly (?), so I would just drop this special case. This is not performance-critical part, this is done once per mailed email. And short, clear and correct code (with fewer special cases) matters more here.